lib/logflare/google/bigquery/event_utils/event_utils.ex

```diff
@@ -1,5 +1,20 @@
 defmodule Logflare.Google.BigQuery.EventUtils do
   @moduledoc false
+
```

Comment:
just a note, we want to move this module into the BigQueryAdaptor namespace eventually, so using defdelegate in BigQueryAdaptor would be ideal. there should be some examples of this in the module already.